sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -283,15 +283,15 @@ def acquire_lease(self, lease_id=None, **kwargs):
 
         .. admonition:: Example:
 
-            .. literalinclude:: ../samples/blob_samples_common.py
-                :start-after: [START acquire_lease_on_blob]
-                :end-before: [END acquire_lease_on_blob]
+            .. literalinclude:: ../samples/file_samples_client.py
+                :start-after: [START acquire_and_release_lease_on_file]
+                :end-before: [END acquire_and_release_lease_on_file]
                 :language: python
-                :dedent: 8
-                :caption: Acquiring a lease on a blob.
+                :dedent: 12
+                :caption: Acquiring a lease on a file.
         """
         lease = ShareLeaseClient(self, lease_id=lease_id)  # type: ignore
-        lease.acquire(**kwargs)
+        lease.acquire(lease_duration=-1, **kwargs)
```

Comment:
Do we not support a user passing in this value?

Reply:
We do not support it for files. The file lease duration can only be infinite while in share leases it can be set